.gitlab-ci.yml

```diff
@@ -353,6 +363,17 @@ variables:
   tags:
     - shell
 
+.build_pf_img_iso_job:
+  stage: build_pf_img
+  environment:
+    name: sourceforge
+    url: ${SF_REPO_URL}/${CI_COMMIT_REF_SLUG}
```

Comment:
This URL will be wrong for a release (tag) because we use ${CI_COMMIT_TAG} later.
This has a minimal effect

Reply:
I added logic to use the tag for releases and the ref for devel related builds